standalone/src/models/custom-application/custom-application/index.ts

```diff
@@ -1,5 +1,17 @@
-export * as CustomApplicationDraft from './custom-application-draft';
+import * as constants from '../constants';
+import { GraphqlModelBuilder } from './builders';
+import * as CustomApplicationPresets from './presets';
 
-export { default as random } from './builder';
-export { default as presets } from './presets';
-export * from './types';
+export const CustomApplicationGraphql = {
+  random: GraphqlModelBuilder,
+  presets: CustomApplicationPresets.graphqlPresets,
+  constants,
+};
+
+/**
+ * @deprecated Use `CustomApplicationGraphql` exported model instead of `CustomApplication`.
+ */
+export const CustomApplication = {
```

Comment:
Please add the `constants` property to this model as well.

Reply:
fixed in https://github.com/commercetools/test-data/pull/827/commits/f4020751f78a48bbcd1f514f6a08c47e0149d50c